src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java

```diff
@@ -833,4 +835,24 @@ protected String getArchiveType()
         return archiveType;
     }
 
+    @Override
+    protected Date convertOutputTimestamp( Date outputTimestamp )
```

Comment:
The name is misleading. You cannot call it conversion if you put in a date and get a date back. `normalizeOutputTimestamp` makes more sense here.

Reply:
renaming to normalizeLastModifiedDate(...) and adding comments to explain the intent of this method